### DISCUSSION TITLE
160

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#106] [General]
# 潮流周刊第160期 - 爱吃炒饭

封面图来源于在外吃饭拍的炒饭图，我发现炒饭能够很好衡量一个餐馆水平和追求的方式，是否粒粒分明、鸡蛋葱花搭配合理、端出来还有锅气，做好了简直属于人间美食。

https://weekly.tw93.fun/posts/160-%E7%88%B1%E5%90%83%E7%82%92%E9%A5%AD

**Comment by vobile-wangzhenfeng** [comment]
然鹅新年第一周还没结束，就开始了RV大战

**Reply by tw93** [reply]
哈哈哈哈哈 我就笑笑不说话

**Comment by xxholly32** [comment]
我也不喜欢
